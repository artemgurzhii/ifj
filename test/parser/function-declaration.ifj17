declare function factorial (foo as integer, bar as integer) as integer
