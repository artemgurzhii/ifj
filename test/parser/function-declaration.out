(function factorial
  (args
    (decl
      (id foo)
      : (id integer)))
    (decl
      (id bar)
      : (id integer)))
  )
  -> (id integer))
